share/doc/homebrew/Interesting-Taps-&-Branches.md

```diff
@@ -56,7 +56,7 @@ You can be added as a maintainer for one of the Homebrew organization taps and a
 
 *   [mistydemeo/tigerbrew](https://github.com/mistydemeo/tigerbrew): Experimental Tiger PowerPC version
 
-*   [homebrew/linuxbrew](https://github.com/Homebrew/linuxbrew): Experimental Linux version
+*   [Linuxbrew/brew](https://github.com/Linuxbrew/brew): Experimental Linux version
 
 *   [nddrylliog/homebrew-mingw](https://github.com/nddrylliog/homebrew-mingw): An experimental port of Homebrew for Windows (with an MSYS/MinGW environment).
```

Comment:
We could consider dropping this. The last commit in that repository is almost three years ago.